src/components/Acknowledgment/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import * as styles from "./Acknowledgment.module.scss";
+import { AiFillLinkedin, AiFillHome } from "react-icons/ai";
+import { SiFiverr } from "react-icons/si";
+
+export default function Acknowledgment({ array, acknowledgmentType }) {
+  return (
+    <div className={styles.acknowledgment_container}>
+      <div className={styles.label}>
+        {acknowledgmentType}
+        {array.length > 1 ? "s" : ""}
+      </div>
+      <div className={styles.names_container}>
+        {array.map((ele1, idx) => {
+          return (
+            <div key={idx} className={styles.name}>
```

Comment:
The reason why the entire name isn't a link is because right now programmatically, the code allows multiple links. IOW, an author or an artist can have more than one link.

All that said, I do agree that the icons are teeny. I'd be able to make it bigger if I could figure out how to center the damn icon. (When I used flex, it'd take up the entire row when it line breaks)